### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by mimansha-swarup**
- homepage
- video Page
- toast

https://user-images.githubusercontent.com/69811810/161412771-8b4f9136-4732-4391-8e3f-2a8ece59186c.mp4

### CODE REVIEW

File: src/Components/Toast/Toast.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { MdCancel } from "react-icons/md";
+const Toast = ({ type, message }) => {
+
+  const [dispay, setDisplay] = useState("flex");
+  const toastType = {
+    success: {
```
Comment (kiranpratyush) [inline src/Components/Toast/Toast.js:7]: We can store it in a separate file

File: src/Page/Home/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  useEffect(() => {
+    (async () => {
+      setStatus({ isLoading: true, error: "" });
+      const response = await axios.get(categoriesApi);
+      if (response.status === 200) {
+        setCategoriesData(response.data.categories);
+        setStatus({ isLoading: false });
+      }
+      try {
+      } catch (error) {
```
Comment (kiranpratyush) [inline src/Page/Home/HomePage.js:20]: I think you have forgot to wrap try

File: src/backend/db/categories.js [hunk 3]
```diff
@@ -1,27 +1,32 @@
 import { v4 as uuid } from "uuid";
+import {categoriesImage} from "../../Assets/index"
 
 /**
  * Category Database can be added here.
```
Comment (kiranpratyush) [inline src/backend/db/categories.js:5]: Can be removed

**Review by kiranpratyush — APPROVED** [review]
Great work Nice UI

File: src/backend/db/categories.js (hunk 3 above)
Comment (mimansha-swarup) [reply src/backend/db/categories.js:5]: i have images locally so need that

File: src/Page/Home/HomePage.js (hunk 2 above)
Comment (mimansha-swarup) [reply src/Page/Home/HomePage.js:20]: are thanks

**Review by ayush65 — APPROVED** [review]
code looks clean 
Ui looks good 
Class Names are self Explanatory

**Pull request merged**